monad-eth-txpool-ipc/src/lib.rs

```diff
@@ -100,7 +95,17 @@ impl EthTxPoolIpcStream {
 
                     let events = bincode::serialize(&events).expect("txpool events are serializable");
 
-                    stream.send(events.into()).await?;
+                    match tokio::time::timeout(std::time::Duration::from_secs(1), stream.send(events.into())).await {
```

Comment:
so the stream is read in the same task (e.g right above this)? so if we await here, that endpoint should become ready?

i am trying to understand when it actually can timeout without failure, e.g if stream is closed i would expect `stream.send(events.into()).await?;` to exit with error